src/lazy/expanded/macro_table.rs

```diff
@@ -100,7 +112,7 @@ impl Macro {
 #[derive(Debug, Clone, PartialEq)]
 pub enum MacroKind {
     Void,
-    Values,
+    ExprGroup,
```

Comment:
As you'll see below, `values` is now a simple `TemplateMacro` that uses `ExprGroup`.